libkbfs/bcache.go

```diff
@@ -42,6 +42,9 @@ type BlockCacheStandard struct {
 
 	bytesLock       sync.Mutex
 	cleanTotalBytes uint64
+
+	// Covers the `prefetchStatus` field for all `blockContainer`s.
```

Comment:
Doesn't this interfere with other locks in `cleanTransient`? Blocks can be removed/overwritten even while this lock is held.

Reply:
I don't understand what you mean by "interfere".  It's not taken in the `onEvict` handler, so I don't think there's any chance of deadlock.  And this isn't trying to cover blocks being removed or overwritten.  It's only to protect against a data race on the `prefetchStatus` field of each individual `blockContainer` struct in the cache.

Reply:
OK sounds good to me.